projects/client/src/lib/components/buttons/popup/_internal/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const POPUP_Z_INDEX = 777;
```

Comment:
Why don't we have this as an actual css variable?

I was actually thinking of creating a `layers.css` file in which we extract all `z-index` values in order to reference them later on.

So my question is why do we have it as a JS constant and later bind it as a css var? 😕